docs/Routes.md

```diff
@@ -270,3 +271,31 @@ fastify.get('/it', { config: { output: 'ciao mondo!' } }, handler)
 
 fastify.listen(3000)
 ```
+
+<a name="version"></a>
+### Version
+If needed you can provide a version option, which will allow you to declare multiple versions of the same route. The versioning should follow the [semver](http://semver.org/) specification.<br/>
+Fastify will automatically detect the `Accept-Version` header and route the request accordingly (advanced ranges and pre-releases currently are not supported).<br/>
+*Be aware that using this feature will cause a degradation of the overall performances of the router.*
```

Comment:
why?

Reply:
Because we spending time analyzing the `Accept-Version` header? :D

Reply:
> Because we spending time analyzing the Accept-Version header? :D

Of course, could you write that reason in docs?

Reply:
I don't think the user should need to know why. It just need to know that the performances will be lower.